tests/testMaskedImageIO.py

```diff
@@ -74,9 +74,15 @@ def setUp(self):
         # Ideally we'd use the standard dictionary and a non-standard file, but
         # a standard file's what we have
         #
-        mask = afwImage.MaskU()
+        self.Mask = afwImage.MaskU
+        mask = self.Mask()
 
-        mask.clearMaskPlaneDict()
+        # Store the default mask planes for later use
+        maskPlaneDict = self.Mask().getMaskPlaneDict()
+        self.defaultMaskPlanes = sorted(maskPlaneDict, key=maskPlaneDict.__getitem__)
+
+        # reset so tests will be deterministic
+        self.Mask.clearMaskPlaneDict()
         for p in ("ZERO", "BAD", "SAT", "INTRP", "CR", "EDGE"):
```

Comment:
No, these mask planes are different from the default and defined purely for this test, which is the reason this was causing the other tests to fail using pytest.